### PR TITLE
update dataset name; allow test datasets

### PR DESCRIPTION
**Pull request by solomon-negusse**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: the dist alerts endpoint has dataset name hardcoded and prevents visualizing test datasets

## What is the new behavior?

- Added flexibility to visualize test datasets
## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: terraform/modules/content_delivery_network/main.tf [hunk 1]
```diff
@@ -455,6 +455,31 @@ resource "aws_cloudfront_distribution" "tiles" {
     }
   }
 
+  ordered_cache_behavior {
```
Comment (danscales) [inline terraform/modules/content_delivery_network/main.tf:459]: Comment at line 426 is confusing.  Should that comment be here right above line 458, and the comment at line 426 should be "Send all cog requests to the tile cache app"?

**Review by danscales — APPROVED** [review]
Looks good!  The alternative routing will be great for further testing of dist alerts.

File: terraform/modules/content_delivery_network/main.tf (hunk 1 above)
Comment (solomon-negusse) [reply terraform/modules/content_delivery_network/main.tf:459]: Both are titiler endpoints. I have added clarifying comment for the DIST alerts one I just added and updated the generic one.

**Comment by solomon-negusse** [comment]
> Looks good! The alternative routing will be great for further testing of dist alerts.

Thanks for the quick review, @danscales.

**Pull request merged**